annyang.js

```diff
@@ -128,6 +128,15 @@
     invokeCallbacks(callbacks.resultNoMatch, results);
   };
 
+  var logMessage = function(logMsg, emphStyle) {
+    emphStyle = emphStyle || debugStyle;
+    if (/%c/.test(logMsg)) {
```

Comment:
Why this check? Won't this mean we can no longer pass a second argument to logMessage (like we did in line 118 before the change)